rs/ledger_suite/icrc1/ledger/Cargo.toml

```diff
@@ -35,6 +35,7 @@ icrc-ledger-types = { path = "../../../../packages/icrc-ledger-types" }
 num-traits = { workspace = true }
 serde = { workspace = true }
 serde_bytes = { workspace = true }
+leb128 = "0.2.5"
```

Comment:
Since `leb128` is now a regular dependency, I think it could be removed from the `[dev-dependencies]` section further down. Additionally, since it's also being introduced in `packages/icrc-ledger-agent/Cargo.toml`, a further optimization could be to add it to the root `Cargo.toml` and use `{ workspace = true }` here.